app/models/concerns/file_importer.rb

```diff
@@ -38,7 +38,12 @@ def import_supervisors
           .split(",")
           .map { |email| User.find_by(email: email.strip) }
           .compact
-        user.volunteers << volunteers if volunteers.present?
+
+        volunteers.each do |volunteer|
+          if !volunteer.supervisor
+            user.volunteers << volunteer
```

Comment:
I want `user` to be renamed to `supervisor` (in a later PR is fine)

Reply:
I will make a TODO comment, thanks!